package.json

```diff
@@ -98,10 +100,12 @@
     "handlebars": "4.7.7",
     "jest": "29.6.1",
     "jest-environment-jsdom": "29.6.1",
+    "npm": "^9.8.1",
```

Comment:
I am not sure at which step this is installed, let me try to remove it.

Reply:
npm definitely should not be a dependency here